src/v1/services/tenantBoardService.ts

```diff
@@ -0,0 +1,33 @@
+import { TenantBoard } from '../models/tenantBoard';
+import { AppDataSource } from '../config';
+import { Optional } from 'sequelize';
+import { UpdateTenantBoard } from '../types/TenantBaord';
+
+//create service for tenant
+export const createTenantBoard = async (req: Optional<any, string> | undefined) => {
+  const transact = await AppDataSource.transaction();
+  const insertTenantBoard = await TenantBoard.create(req, { transaction: transact });
+  await transact.commit();
+  return insertTenantBoard;
+};
+
+//get Single tenant
+export const getTenantBoard = async (tenant_name: string) => {
+  return TenantBoard.findOne({ where: { tenant_name }, raw: true });
+};
+
+//bulk create tenant board
+export const bulkCreateTenantBoard = async (req: any) => {
+  const transact = await AppDataSource.transaction();
+  const insertTenantBoard = await TenantBoard.bulkCreate(req, { transaction: transact });
```

Comment:
check if create with array can do insert many

Reply:
no create use object ,but in bulk create if i give single array of object also possible .

Reply:
so we are gng with using bulk create to create single entry?